index.js

```diff
@@ -0,0 +1,166 @@
+// creates an array from third argument onwards
+const Args = process.argv.slice(2);
+
+//initialize height and width
+let measures;
+const minHeight = 5;
+const minWidth = 19;
+let height;
+let width;
+
+//initialize color choices
+let hueChoice = '';
+let luminosityChoice = '';
+
+if (/\d*x\d*/.test(Args[0]) === true) {
+  // test if first argument matches Regex (digtsxdigits)
+  measures = process.argv[2].split('x'); //generate array from string
+  height = parseInt(measures[0]);
+  width = parseInt(measures[1]);
+  if (height < minHeight) {
+    height = minHeight;
+    console.log(`Too small, I set the height to ${minHeight}.`);
+  }
+  if (width < minWidth) {
+    width = minWidth;
+    console.log(`Too small, I set the height to ${minWidth}.`);
+  }
+  hueChoice = Args[1]; //saves hue choice, first user supplied argument
+  luminosityChoice = Args[2];
+} // saves luminosity choice, second user supplied argument
+
+//if the first argument is ask, ask for input:
+else if (Args[0] === 'ask') {
+  hueChoice = readline.question('Enter a color: ');
+  luminosityChoice = readline.question('Enter a hue (light/dark): ');
+  if (
+    allowedInput.color.includes(hueChoice) === false ||
+    allowedInput.hue.includes(luminosityChoice) === false
+  ) {
+    console.log(
+      `These are not valid arguments. Please use Color: ${allowedInput.color} and Hue: ${allowedInput.hue} `,
+    );
+  }
+} else {
+  hueChoice = Args[0]; //saves hue choice, first user supplied argument
+  luminosityChoice = Args[1];
+} // saves luminosity choice, second user supplied argument
+
+userSelectedColorAsHex = randomcolor({
+  luminosity: luminosityChoice,
+  hue: hueChoice,
+}); // converts color name into hex value
+
+if (userSelectedColorAsHex !== undefined) {
+  //checks if a valid color was entered by the user
+  colorChoice = userSelectedColorAsHex; // updates color Choice
+} else {
+  console.log(
+    'Not a valid color. First arguments needs to be a hue (red/green/blue), second argument needs to be a luminosity (light/dark). I picked a random color for you. Hope you like it:',
+  );
+}
+
+//define function that console.logs a square of 30x10 # in a random color
+// width defined via creating an new array with width
+function makeSquare(
+  width = 30,
+  height = 10,
+  symbol = '#',
+  color = colorChoice,
+) {
+  const arrLine = new Array(width).fill(symbol);
+  const line = arrLine.join('');
+  for (let i = 0; i < height; i++) {
+    console.log(chalk.hex(color)(line));
+  }
+}
+
+//define function that console.logs a square of 30x10 # in a random color
+// width defined via appending to a string
+function makeSquareViaString(
+  width = 30,
+  height = 10,
+  symbol = '#',
+
+  color = colorChoice,
+) {
+  let line = '';
+  for (let i = 0; i < width; i++) {
+    line += symbol;
+  }
+  for (let i = 0; i < height; i++) {
+    console.log(chalk.hex(color)(line));
+  }
+}
```

Comment:
I would remove any code that you're not using right now - it is always in the Git history in case you want to bring it back.